library/src/main/java/com/evernote/android/job/JobIds.java

```diff
@@ -0,0 +1,16 @@
+package com.evernote.android.job;
+
+public class JobIds {
```

Comment:
Could you rename the class to `JobIdsInternal`? I want to avoid that users think that this is an important class. Also please add some JavaDoc.